package.json

```diff
@@ -21,34 +21,35 @@
   "author": "Alex Navasardyan, Ben Holmes",
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.2",
-    "ember-ajax": "0.7.1",
-    "ember-cli": "2.5.0",
+    "broccoli-asset-rev": "^2.4.5",
+    "ember-ajax": "^3.0.0",
+    "ember-cli": "2.13.1",
```

Comment:
In order to get tests working (specifically the ability to use `Ember.onerror` directly), I needed to upgraded ember-qunit. The most sensible path there seemed to upgrade ember-cli and friends.